docs/sources/tempo/setup/operator/object-storage.md

```diff
@@ -43,6 +41,72 @@ Tempo Operator supports [AWS S3](https://aws.amazon.com/), [Azure](https://azure
         type: s3
   ```
 
+### Secure Token Service (STS) Installation
```

Comment:
Is this the AWS Security Token Service? If so, we should match the [AWS STS documentation](https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html) usage:  AWS Security Token Service (STS) on the first occurrence and then AWS STS after that. 
```suggestion
###  AWS Security Token Service (STS) Installation
```